fhirr4/native/src/main/java/io/ballerinax/fhirr4/HTTPToFHIRAdaptor.java

```diff
@@ -133,6 +135,19 @@ public static Object getResourceMethod(BObject service, BArray path, BString acc
         return getResourceMethod(serviceType, pathList, accessor.getValue());
     }
 
+    public static Object isHavingPathParam(ResourceMethodType resourceMethod) {
+
+        String[] paths = resourceMethod.getResourcePath();
+        boolean status = false;
+        for (String s : paths) {
+            if (s.equals(PATH_PARAM_IDENTIFIER)) {
+                status = true;
+                break;
```

Comment:
directly `return true` - no need to use a `status` variable.

Reply:
fixed